tensorflow/lite/micro/tools/make/targets/stm32f4_makefile.inc

```diff
@@ -62,35 +62,24 @@ ifeq ($(TARGET), stm32f4)
   TEST_SCRIPT := tensorflow/lite/micro/testing/test_stm32f4_binary.sh
   # TODO, non working tests.. the micro_speech example partly works
   # TODO(b/158324045): Examine why some tests fail here.
+
   EXCLUDED_TESTS := \
     tensorflow/lite/micro/micro_interpreter_test.cc \
     tensorflow/lite/micro/micro_allocator_test.cc \
     tensorflow/lite/micro/memory_helpers_test.cc \
     tensorflow/lite/micro/memory_arena_threshold_test.cc \
     tensorflow/lite/micro/recording_micro_allocator_test.cc \
-    tensorflow/lite/micro/kernels/logistic_test.cc \
```

Comment:
What exactly requires these to be excluded?

Reply:
don't build with the stn32f4 target. exactly why I'm not sure but we have the same issue for bluepill and I'm guessing that it is because of micro_optional_debug tools which pulls in std::vector, printf ...